examples/scene/prefab.rs

```diff
@@ -0,0 +1,123 @@
+use bevy::{ecs::schedule::ShouldRun, prelude::*, reflect::TypeRegistry, scene::SceneSpawnError, utils::HashMap};
+/// Basic Prefab support for Bevy 0.5.
+/// To use the factory, spawn a component using Commands or World, and the Handle<DynamicScene> of the prefab.
+/// Then add to the factory queue. When the scene is loaded, 
+/// the components of the first entity listed in the file will be added to the entity.
+
+fn setup(
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    mut prefab_factory: ResMut<PrefabFactory>,
+) {
+    let handle: Handle<DynamicScene> = asset_server.load("my_scene/path.scn.ron");
+
+    let prefab_entity = commands.spawn().insert(TrackingComponent).id();
+
+    prefab_factory.add_to_queue(prefab_entity, handle);
+}
+
+pub struct TrackingComponent;
+
+#[derive(Reflect, Clone)]
+#[reflect(Component)]
+struct A {
+    message: String,
+}
+
+impl Default for A {
+    fn default() -> Self {
+        A { message: "Hello!".to_string() }
+    }
+}
+#[derive(Reflect, Clone, Default)]
+#[reflect(Component)]
+struct B {
+    data: Vec<usize>,
+}
+
+fn locate_prefab(
+    query: Query<(&A, &B), (Added<A>, With<TrackingComponent>)>,
+) {
+    for (comp_a, comp_b) in query.iter() {
+        println!("{}, {:?}", comp_a.message, comp_b.data);
+    }
+}
+
+fn main() {
+    let mut app = App::build();
```

Comment:
In order to make this runnable, we should probably have `MinimalPlugins` to generate a game loop, then try loading, spawning and generating some entities.